frontend/src/components/pages/details/Details.tsx

```diff
@@ -569,6 +544,30 @@ export const DetailsUIWithoutContext: React.FC<Props> = ({ slug, parentId, langu
                       );
                     }
 
+                    // Custom HTML templates
+                    if (section.template) {
+                      return (
+                        <section
+                          key={section.name}
+                          ref={sectionRef[section.name]}
+                          id={`details_${section.name}_ref`}
+                        >
+                          <DetailsSection
+                            htmlId={`details_${section.name}`}
+                            titleId={`details.${section.name}`}
+                            className={marginDetailsChild}
+                          >
+                            <HtmlParser
+                              template={section.template}
+                              id={details.id.toString()}
+                              type="trek"
+                              cityCode={details.cities_raw[0]}
```

Comment:
On the deleted lines, there's a `details.cities_raw?.[0]` but not here, `cities_raw` being undefined is not possible anymore?